libs/sdk-ui-charts/src/highcharts/adapter/Chart.tsx

```diff
@@ -95,16 +95,21 @@ export class Chart extends React.Component<IChartProps> {
 
     public createChart(config: HighchartsOptions): void {
         const chartConfig = config.chart;
-        this.chart = new Highcharts.Chart(
-            {
-                ...config,
-                chart: {
-                    ...chartConfig,
-                    renderTo: this.chartRef,
+        try {
+            this.chart = new Highcharts.Chart(
+                {
+                    ...config,
+                    chart: {
+                        ...chartConfig,
+                        renderTo: this.chartRef,
+                    },
                 },
-            },
-            this.props.callback,
-        );
+                this.props.callback,
+            );
+        } catch (error) {
+            // eslint-disable-next-line no-console
+            console.error("Chart could not be rendered with the current config.", error);
+        }
```

Comment:
Without this try/catch, the rendering gets into infinite loop and locks the browser for some invalid HighCharts configs. This never happened before because our configs were aways correct).